README.md

```diff
@@ -82,21 +445,215 @@ Goal 5 partially met what we desired.
 For goal 5, our goal was to allow the user to input a dataset with the same column names, but allow them to be missing a few non-important columns. As of now, this is not possible, however we do allow the user to input identically formatted datsets.
 
 # Testing
-## JS tests
-When compiling the docker container, our js compilation test runs to ensure that our frontend can be successfully copiled
+* Use F12 to check logging and frontend testing status.
+    * Errors are logged as warnings on the console
+    * Nonimplementation issuese are logged as errors, hinting further development.
+    * There are several .db files already present on the `$BACKEND/db` folder and .csv files present on the `/notes/` folder; some of them deliberately anomalous to trigger exception-handling behavior.
+* Backend tests are verified by GitHub actions.
+    * They are to ensure backend operations match those as obtaining data through conventional .xlsx method.   
+    * Rearranging the fields is OK, but removing columns might not be OK
+* Various datasets, including those with anomalous data, have been provided for testing purposes and verifyin that the application handles anomalies correctly.
+    * A reference of the 150 fields is provided below.
+    * More example anomalies are listed on the following section.
+* When compiling the docker container, our JS compilation test runs to ensure that our frontend can be successfully compiled
 
-## Python tests
-To run unit tests, attach an interactive shell to the previously built docker repository.
-To do that, run the following command
-```
-docker run -it <container sha> bash
-```
-To run our testing, run the following shell script
-```
-./run_tests.sh
-```
-## Code linting
-A code linter runs to ensure that our code meets industry standards
+## The 150 fields
```

Comment:
We don't need these fields listed on readme

Reply:
What if someone else want to make said dataset?